src/index.ts

```diff
@@ -27,7 +27,8 @@ function loader(this: interfaces.Webpack, contents: string) {
     }
 
     const rawFilePath = path.normalize(this.resourcePath);
-    const filePath = utils.appendTsSuffixIfMatch(options.appendTsSuffixTo, rawFilePath);
+    let filePath = utils.appendTsSuffixIfMatch(options.appendTsSuffixTo, rawFilePath);
```

Comment:
They can use it together if they want (declare at a single place and then reuse for example), i can code it more generic but after consider backward-compability i chose this method.